Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs

```diff
@@ -28,20 +34,157 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(int[] protein, int[] carbs, int[] fat, string[] dietPlans, int[] expected)
-        {
-            var result = SelectMeals(protein, carbs, fat, dietPlans).SequenceEqual(expected) ? "PASS" : "FAIL";
+        private static void Test(int[] protein, int[] carbs, int[] fat, string[] dietPlans, int[] expected)
+        {
+            nutrition = new Nutrition(protein, carbs, fat);

```

Comment:
If the class nutrition is declared the way given above 
we can have a list of nutriotion objetcs here 
Please read of array vs List and when to use what.
If you are going with the Array
we can have array of nutrition